platforms-serverless/netlify-cli/test.sh

```diff
@@ -3,9 +3,9 @@
 set -eux
 
 if [ "$PRISMA_CLIENT_ENGINE_TYPE" == "binary" ]; then
-  BINARY_STRING=',"files":["index-browser.js","index.d.ts","index.js","query-engine-rhel-openssl-1.0.x","runtime","schema.prisma"]'
+  BINARY_STRING=',"files":["index-browser.js","index.d.ts","index.js","query-engine-rhel-openssl-1.0.x","package.json","runtime","schema.prisma"]'
```

Comment:
That alphabetic order got you again 😆 
```suggestion
  BINARY_STRING=',"files":["index-browser.js","index.d.ts","index.js","package.json","query-engine-rhel-openssl-1.0.x","runtime","schema.prisma"]'
```

Reply:
Dammit, I should've paid more attention to ABC song in school :D